index.rst

```diff
@@ -70,25 +108,43 @@ Each of these configuration repositories are organized as follows:
     #. Site-specific Configuration: ``_summit.yaml``, ``_ncsa.yaml``, ``_base.yaml`` etc.
         - This **optional** file contains contain site specific configuration parameters such as IP addresses and ports.
           Many CSCs have site specific files.
+          SalObj determines which site-specific file should be loaded automatically by parsing the ``LSST_DDS_PARTITION_PREFIX`` environment variable
           Between this file and the ``_init.yaml`` file, **the configuration must be fully defined**
 
     #. Configuration overrides: ``filename.yaml``
         - These **optional** files, referred to as configuration overrides, are only to be used when the values declared in the previous files require changes.
-          These files are loaded manually by the users as is demonstrated in the section-configuration-interaction_.
+          These files are loaded manually by the users as is demonstrated in the `Selecting an Override Configuration_` section.
 
 - If a value is specified in more than one of these files, the most recently seen value is used.
   This means that values in the site-specific (``_<site>.yaml``) file override values in the initial file (``_init.yaml``).
   Also, values in the override file (``filename.yaml``) override values populated in the ``_init.yaml`` and ``_<site>.yaml`` files.
 
-For more technical details about how CSCs handle configurations, see `tstn-017 <https://tstn-017.lsst.io>`__.
+Configuration Files for Unit Tests
+----------------------------------
+
+Unit or integration tests requiring specific information shall utilize an override file that is specific to the test.
+Unit tests utilize configuration files stored in the ``tests/data/config`` directory, as is done for the `ATDome CSC <https://github.com/lsst-ts/ts_ATDome/tree/develop/tests/data/config>`_.
```

Comment:
I'm having trouble telling if this location for unit test configs is a suggestion or a requirement. Please consider something like "Unit tests often utilize" if it is a gentle recommendation, or "should utilize" or "must utilize" if it is a strong recommendation or requirement.

Reply:
I believe it is a requirement from the implementation standpoint, so I'll use "must utilize" since "shall" is reserved for requirements documents.